DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/PostLearningAssessmentViewModelTests.cs

```diff
@@ -250,163 +250,77 @@ public void Post_learning_assessment_can_have_no_next_section()
-                otherItemsInSectionExist: true
-            );
-
-            // When
-            var postLearningAssessmentViewModel =
-                new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
-
-            // Then
-            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().BeFalse();
-        }
-
-        [Test]
-        public void Post_learning_assessment_should_be_only_item_in_only_section()
+        }
+
+        [TestCase(false, false, true)]
+        [TestCase(false, true, false)]
+        [TestCase(true, false, false)]
+        [TestCase(true, true, false)]
+        public void Post_learning_assessment_should_have_onlyItemInOnlySection(
+            bool otherSectionsExist,
+            bool otherItemsInSectionExist,
+            bool onlyItemInOnlySection
+        )
         {
             // Given
             var postLearningAssessment = PostLearningAssessmentHelper.CreateDefaultPostLearningAssessment(
-                otherSectionsExist: false,
-                otherItemsInSectionExist: false
+                otherSectionsExist: otherSectionsExist,
+                otherItemsInSectionExist: otherItemsInSectionExist
             );
 
             // When
             var postLearningAssessmentViewModel =
                 new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
 
             // Then
-            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().BeTrue();
+            postLearningAssessmentViewModel.OnlyItemInOnlySection.Should().Be(onlyItemInOnlySection);
         }
 
-        [Test]
-        public void Post_learning_assessment_should_be_only_item_in_this_section_when_no_other_sections_exist()
+        [TestCase(false, true)]
+        [TestCase(true, false)]
+        public void Post_learning_assessment_should_have_onlyItemInThisSection(
+            bool otherItemsInSectionExist,
+            bool onlyItemInThisSection
+        )
         {
             // Given
             var postLearningAssessment = PostLearningAssessmentHelper.CreateDefaultPostLearningAssessment(
-                otherItemsInSectionExist: false
+                otherItemsInSectionExist: otherItemsInSectionExist
             );
 
             // When
             var postLearningAssessmentViewModel =
                 new PostLearningAssessmentViewModel(postLearningAssessment, CustomisationId, SectionId);
 
             // Then
-            postLearningAssessmentViewModel.OnlyItemInThisSection.Should().BeTrue();
+            postLearningAssessmentViewModel.OnlyItemInThisSection.Should().Be(onlyItemInThisSection);
         }
 
-        [Test]
-        public void Post_learning_assessment_should_not_be_only_item_in_this_section_when_other_sections_exist()
+        [TestCase(false, false, true, true)]
+        [TestCase(false, false, false, false)]
+        [TestCase(true, false, true, false)]
+        [TestCase(false, true, true, false)]
```

Comment:
I don't think this covers all cases? I would include all the possible values of the inputs, both to be thorough and to make it read more like a truth table (which I think makes it easier to understand but that might be just me) 🙂

Reply:
I had just converted the individual tests over, but I agree now that test cases are so concise it is simpler to just add them all. Done in ab474fe 🙂